auto_round/auto_quantizer.py

```diff
@@ -191,6 +191,34 @@ def merge_quantization_configs(
             warnings.warn(warning_msg)
 
         return quantization_config
+    
+    @staticmethod
+    def supports_quant_method(quantization_config_dict):
+        AUTO_QUANTIZATION_CONFIG_MAPPING = {
+            "awq": AwqConfig,
```

Comment:
The implementation differs slightly from the original in Transformers:

1 Ensure that importing our auto_quantizer does not affect other quantization methods besides AWQ and GPTQ, such as HQQ.
2 Ensure compatibility with lower versions of Transformers, such as 4.47.